bin/live/pycbc_live_single_trigger_fits

```diff
@@ -198,110 +177,193 @@ for filename in files:
     except IOError:
         logging.info('IOError with file ' + f)
         continue
+
+    # Triggers for this file only:
+    triggers = {}
     with h5py.File(f, 'r') as fin:
         # Open the file: does it have the ifo group and snr dataset?
         for ifo in args.ifos:
             if not (ifo in fin and 'snr' in fin[ifo]):
                 continue
             live_time[ifo] += 8
```

Comment:
Note: I (still) don't like that this is hard-coded, but don't see a way to get this from the files themselves

Reply:
We can put this in in `pycbc_live` if that helps, should be a trivial change.

Reply:
In any case, taking it from the file name is probably better than hardcoding it. The current value will not work for the early-warning search, for example.